iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/storable_function.hpp

```diff
@@ -56,6 +56,7 @@ class storable_function<Capacity, signature<ReturnType, Args...>>
               typename = typename std::enable_if<std::is_class<Functor>::value
                                                      && is_invocable_r<ReturnType, Functor, Args...>::value,
                                                  void>::type>
+    // AXIVION Next Construct AutosarC++19_03-A12.1.4: implicit conversion of functors is intentional
```

Comment:
I would prefer if you replace `@NOLINTJUSTIFICATION` in the next line with `AXIVION Next Construct AutosarC++19_03-A12.1.4 :` and reuse the justification.